deps/media-playback/media-playback/media.c

```diff
@@ -377,12 +377,21 @@ static void mp_media_next_audio(mp_media_t *m)
 		if (!m->a_cb)
 			return;
 
-		audio = malloc(sizeof(struct obs_source_audio));
 
+		audio = malloc(sizeof(struct obs_source_audio));
 		for (size_t i = 0; i < MAX_AV_PLANES; i++) {
 			if (f->data[i]) {
 				audio->data[i] = malloc(f->linesize[0]);
-				memcpy((void*)audio->data[i], f->data[i], f->linesize[0]);
+				if (m->volume < 100) {
```

Comment:
Better to compare to 100.0f to be sure float comparison works correctly.

Reply:
I had one left over of my initial implementation that was using floats for the volume. It's intended to compare to an integer.

Reply:
(good catch)

Reply:
but float was needed for a line below to work correctly. 

**out[j] = m->volume / 100 * in[j];**

**m->volume / 100** always be 0

Reply:
Updated, good catch